bentoml/frameworks/keras.py

```diff
@@ -168,9 +168,13 @@ def pack(self, data, metadata=None):  # pylint:disable=arguments-renamed
             except ImportError:
                 raise InvalidArgument(error_msg)
 
-        self.bind_keras_backend_session()
         self._model = model
         self._custom_objects = custom_objects
+
+    def pack(self, data, metadata=None):  # pylint:disable=arguments-renamed
+
+        self.bind_keras_backend_session()
+        self._pack_without_bind_session(data, metadata)
```

Comment:
In old codes we use `pack` in two contexts, when packing a in-memory model to the artifacts and when loading the model from disk. In latter case we may not want to bind current session because we already create the session/graph when loading the model from disk.